pkgs/applications/version-management/git-and-tools/cgit/pink.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchurl, callPackage }:
+
+callPackage (import ./common.nix rec {
+  pname = "cgit-pink";
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/cgit-pink/snapshot/cgit-pink-${version}.tar.gz";
+    sha256 = "sha256-oL46NWgqi1VqKNEt0QGBWNXbi2l7nOQDZy1aMivcWuM=";
+  };
+
+  # cgit-pink is tightly coupled with git and needs a git source tree to build.
+  # IMPORTANT: Remember to check which git version cgit-pink needs on every
+  # version bump (look for "GIT_VER" in the top-level Makefile).
```

Comment:
We could enforce a check for this and fail if that does not match.

Reply:
That's a really good idea!  I'd like to leave it for follow-up though, since this PR doesn't change how that works.